packages/catalog-process/package.json

```diff
@@ -29,6 +29,7 @@
     "@types/express": "4.17.21",
     "@types/node": "20.12.12",
     "@types/uuid": "9.0.8",
+    "date-fns": "3.6.0",
```

Comment:
devDependency? shouldn't be a normal dependency instead?

Reply:
I leave it as devDep because some catalog process test are using it